test/end-to-end/browser/helpers/complete-risk-assessment.js

```diff
@@ -109,9 +122,13 @@ export const whenPrisonerIsAssessed = (config = defaultAssessmentConfig) => {
 };
 
 export const fullAssessmentRecommendation = (config) => {
+  if (config.prodSmokeTest) {
```

Comment:
'prodSmokeTest'

Smoke tests are not really PROD specific so we should change this name